geometry_msgs/Pose ee_previsit_pose
---
geometry_msgs/Pose[] ee_poses
